platform_umbrella/apps/home_base/priv/repo/migrations/20241104220527_add_soft_delete.exs

```diff
@@ -0,0 +1,14 @@
+defmodule HomeBase.Repo.Migrations.AddSoftDelete do
+  use Ecto.Migration
+  import Ecto.SoftDelete.Migration
+
+  @existing_tables ~w(installations stored_host_reports stored_usage_reports teams teams_roles users)a
```

Comment:
Only installations have all the soft delete columns in the schemas. We should bring all the tables up to finish. Stored usage reports and stored host reports need both a soft and time-based hard delete.